distributed/http/scheduler/api.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import json
+
+from distributed.http.utils import RequestHandler
+
+
+class APIHandler(RequestHandler):
+    def get(self):
+        self.write("API V1")
+        self.set_header("Content-Type", "text/plain")
+
+
+class RetireWorkersHandler(RequestHandler):
+    async def post(self):
+        self.set_header("Content-Type", "text/json")
+        scheduler = self.server
+        try:
+            params = json.loads(self.request.body)
+            workers_info = await scheduler.retire_workers(**params)
+            self.write(json.dumps(workers_info))
+        except Exception as e:
```

Comment:
I would strongly encourage to redefine the interface of the API call and not just pass kwargs through. A primary goal of this API was to offer a more stable API and I'm not willing to couple the function signature to the HTTP API stability

Reply:
Fair enough. Do you have any particular issue with the expected parameters or is this just an issue with coupling? 

I don't see any reason to make the API different to the underlying function, but we could definitely decouple it so that if the underlying function changes the API does not.

Reply:
I do see reason to not even allow all arguments to be passed here. For instance, the kwarg `remove` is something I would consider to be an internal-use-only argument. I don't see a valid use case for any user to control this behavior. Same for `stimulus_id` which should arguably set by the request handler, not the user.